### PR TITLE
relayer fee section + SDK v3.5.0

### PR DESCRIPTION
**Pull request by CleanBread**

### CODE REVIEW

**Review by on47sky — COMMENTED** [review]
we also have to update this guide: https://gem.azuro.org/hub/apps/guides/advanced/live-tutorial/place-a-bet

**Comment by CleanBread** [comment]
> we also have to update this guide: https://gem.azuro.org/hub/apps/guides/advanced/live-tutorial/place-a-bet

358f2ca67c0e8dc9f9e009d532882dd053b48cf5

**Review by on47sky — APPROVED** [review]

File: pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx [hunk 1]
```diff
@@ -29,14 +29,56 @@ if they cannot.
 We recommend you as a frontend operator to create a convenient interface to do that in 1 click and claim
 `1000 AZUSD` to the user's wallet.
 
-### Give a spending approval
+### Relayer fee
 
 In our live betting architecture users don't make transactions, they just sign a message that contains all the bet information and place an order.
-This order then been executed by a [Relayer](/contracts/live-relayer) - special backend service for transactions broadcasting.
+This order then been executed by a [Relayer](/contracts/live-relayer) - special backend service for transactions broadcasting.
```
Comment (Avvrik) [inline pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx:35]: This order is then being executed by....

File: pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx [hunk 2]
```diff
@@ -29,14 +29,56 @@ if they cannot.
...
+
+The `Relayer` uses bet tokens as a fee then spending amount for approval is:
+```ts
+const spendingAmount = relayerFeeAmount + betAmount
+```
 
-<Callout type="feature">
-  While we work in a testnet, you can ask users to give an approval to spend an unlimited amount of tokens.
-</Callout>
+That `spendingAmount` you should to use for spending approval.
```
Comment (Avvrik) [inline pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx:81]: This `spendingAmount` should be used for spending approval.

File: pages/hub/apps/sdk/data-hooks/useLiveBetFee.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { Callout } from 'components'
+
+This hook is used to fetch the relayer fee amount for live bet.
```
Comment (Avvrik) [inline pages/hub/apps/sdk/data-hooks/useLiveBetFee.mdx:3]: ...for live bets.
если для одного, то for a live bet.

File: pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx [hunk 4]
```diff
@@ -29,14 +29,56 @@ if they cannot.
...
 `1000 AZUSD` to the user's wallet.
 
-### Give a spending approval
+### Relayer fee
 
 In our live betting architecture users don't make transactions, they just sign a message that contains all the bet information and place an order.
-This order then been executed by a [Relayer](/contracts/live-relayer) - special backend service for transactions broadcasting.
+This order then been executed by a [Relayer](/contracts/live-relayer) - special backend service for transactions broadcasting. 
+
+Instead of charging a fee in native tokens when a user sends a transaction, the `Relayer` uses bet tokens as a fee.
```
Comment (Avvrik) [inline pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx:37]: ...the `Relayer` uses some of the tokens from the bet amount as a fee.

Comment (CleanBread) [reply pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx]: f939912dae6e66990826b1bf2cdec72d79ecc5d8

File: pages/hub/apps/sdk/data-hooks/useLiveBetFee.mdx (hunk 3 above)
Comment (CleanBread) [reply pages/hub/apps/sdk/data-hooks/useLiveBetFee.mdx]: f939912dae6e66990826b1bf2cdec72d79ecc5d8

File: pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx (hunk 2 above)
Comment (CleanBread) [reply pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx]: f939912dae6e66990826b1bf2cdec72d79ecc5d8

File: pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx (hunk 1 above)
Comment (CleanBread) [reply pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx]: f939912dae6e66990826b1bf2cdec72d79ecc5d8

**Review by Avvrik — APPROVED** [review]

**Pull request merged**